core/src/main/java/tc/oc/pgm/stats/StatsMatchModule.java

```diff
@@ -97,8 +107,12 @@ public StatsMatchModule(Match match) {
     this.match = match;
   }
 
-  public Map<UUID, PlayerStats> getStats() {
-    return Collections.unmodifiableMap(allPlayerStats);
+  public Map<UUID, PlayerStats> getGlobalStats() {
+    return allPlayerStats;
+  }
+
+  public Table<Team, UUID, PlayerStats> getStats() {
+    return Tables.unmodifiableTable(stats);
```

Comment:
keep getStats signature intact (returning global stats) and have getParticipationStats as the new one returning a table, for api backwards-compatibility